### PR TITLE
Add some more other shit

### PR DESCRIPTION
**Pull request by orcachillin**

### CODE REVIEW

File: RosaServer/structs.h [hunk 1]
```diff
@@ -717,7 +718,8 @@ struct VehicleType {
 	float mass;     // 38
 	PAD(0x17878 - 0x38 - 4);
 	int numWheels;  // 17878
-	PAD(0x185C0 - 0x17878 - 4);
+	PAD(99636 - 17878 - 4);
```
Comment (checkraisefold) [inline RosaServer/structs.h]: hex please

File: .gitignore [hunk 2]
```diff
@@ -98,4 +98,5 @@ compile_commands.json
 CTestTestfile.cmake
 _deps
 CMakeUserPresets.json
-RosaServer/git_version.cpp
\ No newline at end of file
+RosaServer/git_version.cpp
+RosaServerSatellite/rosaserversatellite
```
Comment (checkraisefold) [inline .gitignore]: remove this

**Review by checkraisefold — CHANGES_REQUESTED** [review]
make a PR on rosaservercore as well documenting the new properties

File: .gitignore (hunk 2 above)
Comment (orcachillin) [reply .gitignore]: no

**Comment by orcachillin** [comment]
https://github.com/jpxs-intl/RosaServerCore/pull/1

**Review by checkraisefold — APPROVED** [review]
lgtm

**Pull request merged**